packages/amplify-cli-core/package.json

```diff
@@ -33,7 +33,9 @@
     "hjson": "^3.2.1",
     "js-yaml": "^4.0.0",
     "lodash": "^4.17.19",
-    "open": "^7.3.1"
+    "node-fetch": "^2.6.1",
+    "open": "^7.3.1",
+    "semver": "^7.3.5"
```

Comment:
please update the references at other places of `semver` to this version, we use 2 other versions currently.